GitUI/HelperDialogs/FormProcess.cs

```diff
@@ -19,8 +19,15 @@ public partial class FormProcess : FormStatus
         public HandleOnExit? HandleOnExitCallback { get; set; }
         public readonly Dictionary<string, string> ProcessEnvVariables = new();
 
-        private FormProcess(GitUICommands? commands, ConsoleOutputControl? outputControl, ArgumentString arguments, string workingDirectory, string? input, bool useDialogSettings, string? process)
-            : base(commands, outputControl, useDialogSettings)
+        private FormProcess(IServiceProvider serviceProvider,
+                            GitUICommands? commands,
```

Comment:
Why are we doing this? `GitUICommands` is a service provider

Reply:
`commands` is optional (can be `null`), `serviceProvider` is mandatory.